src/main/java/Yukee.java

```diff
@@ -0,0 +1,119 @@
+import java.util.Scanner;
+
+public class Yukee {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        Task[] tasks = new Task[100];
+        int taskCount = 0;
+
+        String logo = """
+                 Y   Y  U   U  K   K  EEEEE  EEEEE
+                  Y Y   U   U  K  K   E      E   \s
+                   Y    U   U  KKK    EEEE   EEEE\s
+                   Y    U   U  K  K   E      E   \s
+                   Y     UUU   K   K  EEEEE  EEEEE
+                """;
+        System.out.println("Hello! I'm Yukee, your friendly assistant!\n" + logo);
+        System.out.println("(✿◠‿◠) What can I do for you today? (◕‿◕)");
+
+        while (true) {
+            String input = scanner.nextLine().trim();
+            String[] inputSplit = input.split(" ", 2);
+            String command = inputSplit[0].toLowerCase();
+
+            switch (command) {
+                case "bye":
```

Comment:
Based on java coding standards, "The switch statement should have the following form: Note there is no indentation for case clauses."
```suggestion
         switch (command) {
         case "bye":
         ...
         }
```